build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
 dependencies {
     compile 'org.apache.maven:maven-model-builder:3.0.4'
+    compile 'commons-codec:commons-codec:1.10'
```

Comment:
I think we can do this with a groovy or JVM Base64 encoder instead of adding another dependency

http://mrhaki.blogspot.com/2009/11/groovy-goodness-base64-encoding.html